runtimeconfig/manager.go

```diff
@@ -147,26 +150,51 @@ func (om *Manager) loop(ctx context.Context) error {
 // loadConfig loads all configuration files using the loader function then merges the yaml configuration files into one yaml document.
 // and notifies listeners if successful.
 func (om *Manager) loadConfig() error {
-	mergedConfig := map[string]interface{}{}
+	rawData := map[string][]byte{}
+	hashes := map[string]string{}
```

Comment:
nit: these could be initialized to the size of `om.cfg.LoadPath`, no?

Reply:
Yes, but values below 8 are ignored in `make(map[]..., len)` call anyway, so why bother.

Reply:
(I mean, most people will use handful of files, hopefully not 8+)